src/main/drivers/serial_uart_impl.h

```diff
@@ -193,11 +121,11 @@ typedef struct uartHardware_s {
 
     rccPeriphTag_t rcc;
 
-#if !defined(STM32F7)
+#if defined(UART_AF_CAPABLE)
     uint8_t af;
```

Comment:
This is the case where AF is common for all pins for given UART port (and IIRC F4 has single AF for all UARTs)